Src/UIComponents/TileMyWeight/TileMyWeight.ts

```diff
@@ -0,0 +1,102 @@
+import { generateWhiteButton } from '../Buttons/Buttons';
+const moment = require('moment');
```

Comment:
isn't typescript throwing some warnings for 'require' here? I had some issues with that :)

Reply:
Yea I'm trying to change it into import but there are some other issues.